keras_model.py

```diff
@@ -0,0 +1,36 @@
+from keras.models import load_model
+from multiprocessing import Lock
+from multiprocessing.managers import BaseManager
+
+import config
+cfg = config.get_localized_config()
+
+class KerasModel():
+    def __init__(self):
+        self.mutex = Lock()
+        self.model = None
```

Comment:
Optional, but I think you could just initialize the model here and skip the `initialize` method.